base/deprecated.jl

```diff
@@ -3423,6 +3423,12 @@ workspace() = error("workspace() is discontinued, check out Revise.jl for an alt
 # PR #25113
 @deprecate_binding CartesianRange CartesianIndices
 
+# Use getproperty instead of getindex for Factorizations
+function getindex(F::Factorization, s::Symbol)
+    depwarn("F[:$s] is deprecated, use F.$s instead.", :getindex)
```

Comment:
Given that this method's `F` may have any name in the caller's code, `F[:...]` might be a bit obscure. Perhaps expand this depwarn a bit? A sketch:

*Extracting factorization components via `getindex(F::Factorization, s::Symbol)` methods, usually written `F[:s]` where `F` is the `<:Factorization` and `:s` identifies the factorization component (for example `aQRfact[:Q]`), has been deprecated in favor of `F.s`.*

Reply:
Missed or disagree? :)

Reply:
Decided to think a little longer and then forgot. I've decided that I prefer the current shorter version. If users later report that it is unclear we can adjust but I wouldn't expect that.